src/components/Carousel.js

```diff
@@ -301,6 +301,7 @@ export default class Carousel extends Component {
    * @param {number} index of the element drag started on
    */
   onMouseDown = (e, index) => {
+    console.log('onMouseDown 23', { e, index });
```

Comment:
Later I can remove debugging code.
I inserted this `console.log` and later modified it in order to check whether the updated build is deployed.